Source/Projections/Builder/CouldNotCreateProjectionOf.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.SDK.Projections.Store;
+using Dolittle.SDK.Tenancy;
+
+namespace Dolittle.SDK.Projections.Builder;
+
+/// <summary>
+/// Exception that gets thrown when an instance of <see cref="ProjectionOf{TReadModel}"/> could not be instantiated for a Tenant and a <see cref="Type"/>.
+/// </summary>
+public class CouldNotCreateProjectionOf : Exception
```

Comment:
I don't think we need this if we construct `ProjectionOf` from the `IProjectionStore` instead of using the Activator.